### DISCUSSION TITLE
CSS and admin panel not loading after install

### DISCUSSION THREAD

**Discussion by alfiegarratt1** [microweber/microweber#980] [Q&A]
Hello, I'd appreciate help with this. Formatting is not loading and the admin panel not accessible doesn't exist.
Here's the image:
![snip2](https://user-images.githubusercontent.com/121064551/208552551-0d9de169-24d6-4605-84ff-88d0e6ea23fe.PNG)

I've downloaded it through composer, web installer and zip, and changed the PHP version with no success. I am running on apache server windows 10 with MySQL.

Thanks!

**Comment by alfiegarratt1** [comment]
PHP Version 8.1.13 (8.2 gave more errors)
Changed session_auto_start to 1 as per researched solutions. Still no avail.

**Comment by teran-s** [comment]
Try installing this with the latest version of XAMPP. You might have to change the php.ini file to enable GD and Intl extensions.

**Comment by markcjf** [comment]
I found MAMP to be better for microweber than XAMPP.
